app/src/main/java/io/apicurio/registry/metrics/LivenessUtil.java

```diff
@@ -17,6 +17,7 @@
 package io.apicurio.registry.metrics;
```

Comment:
@carlesarnal This change should go in a separate PR?

Reply:
No, these changes are needed.